DESCRIPTION

```diff
@@ -8,9 +8,8 @@ Authors@R: c(
 	person("Xiao","Ni", role = "aut"),
 	person("James","Buchanan", role="aut"),
 	person("Susan","Duke", role="aut"),
-	person("Interactive Safety Graphics Working Group", role = "aut"),
-	person("Rho Inc.", role = "cph"))
```

Comment:
I believe we need to list the copyright holder from the LICENSE if its not going to be the maintainers. CRAN will reject a package if the copyright holder isn't noted in the description

Reply:
Thanks for the heads up, @elimillera. I'm going to file a separate issue to deal with this and talk with the group about how to move forward.